about.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="//use.fontawesome.com/releases/v5.0.7/css/all.css"
+    />
+    <title>About Our Digital Jerusalem</title>
+</head>
+<body>
+    <header>
+        <div class="social-nav">
+          <ul class="social-menu">
+            <li class="social-item">
+              <a href="#" class="social-link"><i class="fab fa-twitter"></i></a>
+            </li>
+            <li class="social-item">
+              <a href="#" class="social-link"
+                ><i class="fab fa-facebook-f"></i
+              ></a>
+            </li>
+            <li class="social-item">
+              <a href="#" class="social-link">English</a>
+            </li>
+            <li class="social-item">
+              <a href="#" class="social-link">My Page</a>
+            </li>
+            <li class="social-item">
+              <a href="#" class="social-link">Log Out</a>
+            </li>
+          </ul>
+        </div>
+        <nav class="navbar">
+          <a href="./index.html"
+            ><h1 class="navbar-title">OurDigital Jerusalem</h1></a
+          >
+          <ul class="nav-menu">
+            <li class="nav-item">
+              <a href="./index.html" class="nav-link"
+                ><i class="fas fa-home"></i
+              ></a>
+            </li>
+            <li class="nav-item">
+              <a href="./about.html" class="nav-link">About</a>
```

Comment:
- [ ] Good job adding the home icon above 👍🏿, you can change the about link to the home link so users can easily navigate to the home page on mobile and desktop using this link 👍🏿

Reply:
@prolajumokeoni done